go.mod

```diff
@@ -63,3 +63,5 @@ require (
 )
 
 go 1.21.3
+
+replace github.com/manicminer/hamilton => github.com/MarkDordoy/hamilton v0.17.1-0.20240611151114-899c6ce169f6
```

Comment:
revert?

Reply:
Sorry forgot to note that this depends on https://github.com/manicminer/hamilton/pull/285, will rebase prior to merge